doc/tutorial_regression.rst

```diff
@@ -1,5 +1,5 @@
-Tutorial for regression
-=======================
+Tutorial
+========
```

Comment:
The tutorial is now automatically generated from a notebook, I will add the .md file in PR #119

Reply:
OK